utils/capellacontrol/manager.go

```diff
@@ -62,7 +62,7 @@ func (m *Manager) WaitForClusterState(
 			continue
 		}
 
-		break
+		return nil
```

Comment:
Rather than having this function try and be a 'generic solution' for all types of clusters, we should probably modify this function to either take the type of cluster its dealing with as input, or alternatively have it perform both ListClusters and ListAllColumnars as part of a single loop.  I think there are some edge-cases where it could fall into the wrong loop.

Reply:
Makes sense, it was a bit messy. I've added a columnar param and made into 1 for loop.